packages/FormElement/src/components/Description/Description.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import descriptionStyles from "./Description.styles";
+import DescriptionStyled from "./Description.styles";
```

Comment:
I think the convention we settled on with Danil was to export as `styled`, so if there are multiple, you can just import that.  Then use like `<styled.Description>`.  Kinda trivial difference here, but when you have multiple being exported, it makes consuming easier.

Reply:
Example:
https://github.com/acl-services/paprika/blob/master/packages/Modal/src/Modal.styles.js

Reply:
Oh strange, I already had made that change a day ago, are you looking at current code?

Reply:
Ok, i'll use that approach for the components i've updated

Reply:
Gotta say i'm not all that keen on it though, I think it makes the Markup a bit less readable with `<styled.` written before every component. I'm not so sure it's good for every situation